fe2-android/app/src/main/java/com/github/dedis/popstellar/model/network/method/message/data/election/PlainVote.kt

```diff
@@ -31,14 +32,31 @@ class PlainVote : Vote {
       vote: Int?,
       writeInEnabled: Boolean,
       writeIn: String?,
-      electionId: String?
+      electionId: String?,
   ) {
+    verify()
+        .isNotEmptyBase64(questionId, "question ID")
+        .isNotEmptyBase64(electionId ?: "", "election ID")
```

Comment:
I don't see any reason why we should pass an empty string when it's null rather than just pass directly `electionId`.

If electionId is null in the `Election.generateElectionVoteId` function it will throw the illegal argument exception, so we can safely check if it's null right from the beginning in the MessageValidator

Reply:
I remember I did this because IDE was screaming about it, but it's not anymore... Strange :man_shrugging:  
You are right tho, thanks :+1: 

Also is there a reason we had `String?` and not `String` for electionId? I did not want to touch it without knowing, but in my opinion we can just remove the `?` to ensure we are not given a null argument.